README.md

```diff
@@ -166,6 +166,16 @@ Self Managed calling apps are an advanced topic, and there are many steps involv
 
 ## Methods
 
+### getInitialEvents
+_This feature is available only on iOS._
+
+If there were some actions performed by user before JS context has been created, this method would return early fired events. This is alternative to "didLoadWithEvents" event.
+>>>>>>> getInitialEvents-docs
```

Comment:
Probably an issue with your merge?

Reply:
My bad! This is one is from other PR https://github.com/react-native-webrtc/react-native-callkeep/pull/421 which should be ok I think. If you want to, I can remove those changes